Core/Src/platform.c

```diff
@@ -0,0 +1,9 @@
+#include "platform.h"
+
+uint32_t millis(void) {
```

Comment:
really curious how you got this to compile. Maybe fixed with one of the canlib update and we just need to try again on processor

Reply:
Build file filter